pycaption/sami.py

```diff
@@ -735,13 +733,12 @@ def feed(self, data):
             raise CaptionReadSyntaxError(f'SAMI File contains "{no_cc}"')
 
         # try to find style tag in SAMI
-        try:
-            # prevent BS4 error with huge SAMI files with unclosed tags
-            index = data.lower().find("</head>")
-            style = BeautifulSoup(data[:index], "lxml").find('style')
-            if style and style.contents:
-                self.styles = self._css_parse(' '.join(style.contents))
-        except AttributeError:
+        # prevent BS4 error with huge SAMI files with unclosed tags
```

Comment:
If error handling is no longer necessary, I think the comment about it should go too?

Reply:
In case you're referring to the one just above, from what I understood in the code, this was related to searching the data only up to the index that's set at the closing head tag. So I believe it should actually be kept.
And in the case you're referring to the other one, it should still be valid, since that's the code block searching for the style tag.